### ISSUE TITLE
Question: Work as browser global?

### ISSUE THREAD

**Issue by tegon** [ikr/money-math#6]
Hi, great job here, the library is awesome.
I just wanna know if you have any plans to make this work as a browser global also, I'm thinking in something like underscore does:

```js
if (typeof exports !== 'undefined') {
    if (typeof module !== 'undefined' && module.exports) {
      exports = module.exports = _;
    }
    exports._ = _;
  } else {
    root._ = _;
  }
```
I saw in #3 that this already work with Browserify, but I would like to use it an project that doesn't have any module system. 

If it makes sense, I can help with a PR also. 
Thanks

**Comment by ikr** [comment]
Hi @tegon 

Thanks. Yes, I agree that would be valuable. So, a PR on that is welcome. Please only consider the [jsbn](https://www.npmjs.com/package/jsbn) dependency.

**Comment by tegon** [comment]
@ikr sorry for taking so long on that. Just opened #7 with my solution, let me know your thoughts on it.

**Comment by ikr** [comment]
Implemented in #7
